build/yamls/flow-visibility.yml

```diff
@@ -4681,12 +4682,12 @@ data:
     }
 kind: ConfigMap
 metadata:
-  name: grafana-dashboard-config-h4fg25d7k9
+  name: grafana-dashboard-config
   namespace: flow-visibility
 ---
 apiVersion: v1
 data:
-  dashboard_provider.yml: |
+  dashboard_provider.yaml: |-
```

Comment:
Is this name change from `.yml` to `.yaml` is required by Helm?

Reply:
it is not mandatary by Helm. Helm only requires files like `Chart.yaml` and `values.yaml` to be ended by `.yaml`. I change all the YAML files extension under Helm chart dir to `.yaml` to keep consistency and avoid confusions or errors in future. I can change some of them back if you think this do bring new confusions.

Reply:
Got it, current change sounds good to me for consistency.